lonestar/analytics/cpu/k-shortest-paths/ksssp_cli.cpp

```diff
@@ -156,7 +156,7 @@ main(int argc, char** argv) {
   katana::TxnContext txn_ctx;
 
   auto pg_result =
-      Ksp(pg.get(), startNode, reportNode, edge_property_name, &txn_ctx,
```

Comment:
why remove this? this is required for loading edge weights, I think.

Reply:
edge_weight_property_name was used in two locations for Ksp: [1](https://github.com/dqawami/katana/blob/f16ade5d2d7ad65ec1d606e232d01509b211aa3e/libgraph/src/analytics/k_shortest_paths/ksssp.cpp#L430) and [2](https://github.com/dqawami/katana/blob/f16ade5d2d7ad65ec1d606e232d01509b211aa3e/libgraph/src/analytics/k_shortest_paths/ksssp.cpp#L460), both of which are removed when switching to views

Reply:
no, it is still required for loading edge weights. Check how it is used in the louvain clustering app.

Reply:
here is the example from louvain clustering: https://github.com/uditagar/katana/blob/a0ab4ac12da2e6d20add5a12151a972a4ee894e7/lonestar/analytics/cpu/louvain_clustering/louvain_clustering_cli.cpp#L127

Reply:
In the original implementation of k-shortest-paths, edge_property_name was only used once [here](https://github.com/KatanaGraph/katana/blob/1be615d24e3435e38f456068b1c5eafeae0d07d7/lonestar/analytics/cpu/k-shortest-paths/SSSP.cpp#L319), should I still add it?

Reply:
oh, that was last year's format. Now the app's src file receives the edge property name and then this is passed to the view construction, where it is then loaded. this is already in use in the louvain app's code.